ask-sdk-core/tst/util/RequestEnvelopeUtil.spec.ts

```diff
@@ -43,7 +44,7 @@ describe('RequestEnvelopeUtils', () => {
             templateVersion : '1.0',
             markupVersion : '1.0',
         },
-    };
+    };requestEnvelope.context.System.user.userId = 'mockUserId'
```

Comment:
Could you please write this one in a new line?

Reply:
Thanks for the feedback!
I've fixed it :)